addons/fastroping/CfgVehicles.hpp

```diff
@@ -50,11 +50,40 @@ class CfgVehicles {
                 statement = QUOTE([vehicle _player] call FUNC(stowFRIES));
                 showDisabled = 0;
             };
-            class ACE_deployRopes {
-                displayName = CSTRING(Interaction_deployRopes);
-                condition = QUOTE([ARR_2(_player, vehicle _player)] call FUNC(canDeployRopes));
-                statement = QUOTE([ARR_2(QUOTE(QGVAR(deployRopes)), [vehicle _player])] call CBA_fnc_serverEvent);
+            class ACE_deployRopes12 {
+                displayName = CSTRING(Interaction_deployRopes12);
+                condition = QUOTE((('ACE_rope12' in (backpackItems _player)) || ('ACE_rope12' in (ItemCargo vehicle _player))) && [_player, vehicle _player] call FUNC(canDeployRopes));
+                statement = QUOTE([QGVAR(deployRopes), [vehicle _player, 12]] call CBA_fnc_serverEvent);
                 showDisabled = 0;
+                priority = 1;
+            };
+            class ACE_deployRopes15 {
+                displayName = CSTRING(Interaction_deployRopes15);
+                condition = QUOTE((('ACE_rope15' in (backpackItems _player)) || ('ACE_rope15' in (ItemCargo vehicle _player))) && [_player, vehicle _player] call FUNC(canDeployRopes));
```

Comment:
I'd prefer to move the cheapest check to the start and lazy eval the rest. As this inhibits interaction menu perf.
I'd say put canDeployRopes first. But I don't know how expensive that is compared to the rest.

Reply:
moved all checks to func
I think checking items last should be best as stuff like attack choppers won't use this at all